src/mongocxx/test/collection.cpp

```diff
@@ -2336,7 +2336,7 @@ TEST_CASE("create_index tests", "[collection]") {
 
         bool unique = options.unique().value();
         auto validate = [unique](bsoncxx::document::view index) {
-            auto expire_after = index["expireAfter"];
+            auto expire_after = index["expireAfterSeconds"];
```

Comment:
Did the previous version only work due to the bug being fixed above, or is this change unrelated?

Reply:
Only worked due to the bug.